src/Confluent.Kafka/Structs/Error.cs

```diff
@@ -64,8 +72,19 @@ public override int GetHashCode()
         public static bool operator !=(Error a, Error b)
             => !(a == b);
 
+        // Returns the string representation of the error.
```

Comment:
I'd put the first line in a <summary></summary> block and the second two lines in <reason></reason>